vocode/streaming/agent/state_agent.py

```diff
@@ -330,33 +246,14 @@ async def handle_options(
                 "- Ensure not to suggest any actions or offer alternatives.\n"
             )
             # we do not care what it has to say when switching or continuing
-            output = await call_ai(prompt)
+            output = await call_ai(prompt, {})
             speak(output)
 
             async def resume(human_input):
                 logger.info(
                     f"continuing at {state['id']} with user response {human_input}"
                 )
-                branch_decision, ai_response = await on_user_input_within_state(
-                    state, human_input, last_bot_message, call_ai, logger
-                )
-
-                if branch_decision == BranchDecision.PAUSE:
-                    if ai_response:
-                        speak(ai_response)
-                    return await handle_options(
-                        state=state,
-                        go_to_state=go_to_state,
-                        speak=speak,
-                        call_ai=call_ai,
-                        state_machine=state_machine,
-                        get_chat_history=get_chat_history,
-                        logger=logger,
-                    )
-                if branch_decision == BranchDecision.CONTINUE:
-                    return await go_to_state(get_default_next_state(state))
-                if branch_decision == BranchDecision.SWITCH:
-                    return await go_to_state("start")
+                return await go_to_state(get_default_next_state(state))
```

Comment:
I think we need to go to the _current_ state here, since this code path is for when the user picked the Question edge